lang/cs/Org.Apache.REEF.Network.Examples/GroupCommunication/PipelineBroadcastReduceDriverAndTasks/PipelinedBroadcastReduceDriver.cs

```diff
@@ -124,7 +123,7 @@ public void OnNext(IEvaluatorRequestor evaluatorRequestor)
         public void OnNext(IAllocatedEvaluator allocatedEvaluator)
         {
             IConfiguration contextConf = _groupCommDriver.GetContextConfiguration();
-            IConfiguration serviceConf = _groupCommDriver.GetServiceConfiguration();
+            IConfiguration serviceConf = _groupCommDriver.GetServiceConfiguration(_codecConfig);
```

Comment:
You should call GetServiceConfiguration() without parameter and then merge it with _condecConfig at this line.